<launch>
  <!--launch-prefix="gdb -ex run - -args" -->

  <node name="generate_test_dataset" type="generate_test_dataset" pkg="nbv_3d_cnn" output="screen" clear_params="true" >
    <param name="source_images_prefix" type="string" value="$(find nbv_3d_cnn)/data/scenes/" />
    <param name="source_image_suffix" type="string" value=".bt" />

    <param name="dest_images_prefix" type="string" value="$(find nbv_3d_cnn)/data/environments_3d/" />

    <!--<param name="sub_image_width" type="int" value="256" />-->
    <param name="environment_resize" type="string" value="1 1" />

    <param name="sensor_range_voxels" type="double" value="24" />

    <param name="sensor_resolution_y" type="int" value="24" />
    <param name="sensor_resolution_x" type="int" value="32" />
    <param name="sensor_focal_length" type="double" value="28" /> <!-- FOV ~ 60 deg -->

    <param name="view_cube_resolution" type="int" value="16" />
    <!--<param name="view_cube_resolution" type="int" value="4" />-->

    <param name="mode_3d" type="string" value="3d" />

    <param name="num_view_poses_min" type="int" value="10" />
    <param name="num_view_poses_max" type="int" value="15" />

    <param name="prefix_list" type="string" value="scene_" />
  </node>

</launch>
